vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -1,3 +1,8 @@
+export IRONIC_HOST_IP='{{ IRONIC_HOST_IP }}'
+export '{{ DEPLOY_KERNEL_URL }}'
+export '{{ DEPLOY_RAMDISK_URL }}'
+export '{{ IRONIC_URL }}'
+export '{{ IRONIC_INSPECTOR_URL }}'
```

Comment:
This is why the CI breaks. Same for ubuntu

Reply:
Fixed.